code/modules/atmospherics/FEA_airgroup.dm

```diff
@@ -269,11 +266,6 @@
 				if(connection_difference)
 					if(connection_difference > 0 && !isnull(self_border))
 						self_border.consider_pressure_difference(connection_difference, get_dir(self_border,enemy_tile))
-					else
-						var/turf/enemy_turf = enemy_tile
-						if(!isturf(enemy_turf))
-							enemy_turf = enemy_tile.loc
```

Comment:
why was this removed btw?

Reply:
fair

Reply:
it also did absolutely nothing because it inputted a dir of 0

Reply:
it was a tile pointing to itself

Reply:
i removed it later on

Reply:
Think i can avoid checking if its nonsimulated by making sure that cant ever happen in normal gameplay. replacewith only rebuilds groups if the new tile is simulated but doesnt rebuild if the old tile was simulated and changed into a nonsimulated one. ill do that in a new pr.

Reply:
leaving an assert istype in every for loop could get expensive

Reply:
It already had an istype check, either put the code back or put an assert, don't add a silent failure.

Reply:
putting a stacktrace instead, shouldn't be problematic enough to warrant a crash but will rebuild group. previous code did nothing useful at all since it had a dir of 0 inputted.